<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>917</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>861</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Books</string>
     </attribute>
     <widget class="QPushButton" name="findBooksButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>211</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>511</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="authorEdit">
         <property name="placeholderText">
          <string>Автор</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit">
         <property name="placeholderText">
          <string>Название</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="booksTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>1021</width>
        <height>681</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="addBookButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>110</y>
        <width>241</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить книгу</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Authors</string>
     </attribute>
     <widget class="QLineEdit" name="authorNameEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>509</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Автор</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="authorsTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>1031</width>
        <height>681</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="addAuthorButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>80</y>
        <width>241</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить автора</string>
      </property>
     </widget>
     <widget class="QPushButton" name="findAuthorButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>20</y>
        <width>211</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Users</string>
     </attribute>
     <widget class="QLineEdit" name="userEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>509</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Пользователь</string>
      </property>
     </widget>
     <widget class="QPushButton" name="findUserButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>20</y>
        <width>211</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="usersTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>1031</width>
        <height>751</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Essays</string>
     </attribute>
     <widget class="QTableWidget" name="essaysTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>1031</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addEssayButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>201</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить текст для анализа</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="essayText">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>430</y>
        <width>1031</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tests</string>
     </attribute>
     <widget class="QPushButton" name="addTestButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>981</width>
        <height>291</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Добавить тест</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1031</width>
        <height>211</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Подсказки по добавлению тестов&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;В первой графе напишите ответ или выбор.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Во второй: вопрос. Обязательно последним символом поставьте знак препинания!&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;В третьей: варианты ответа (если нужно). Перечислите их через знак &amp;quot;;&amp;quot;. &lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; text-decoration: underline;&quot;&gt;БЕЗ ПРОБЕЛОВ.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;В четвертой: правильный вариант ответа.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Название теста не должно быть использовано.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Пройти тест Вы можете, если зайдете в систему как пользователь.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
